metadata.rb

```diff
@@ -16,3 +16,4 @@
 supports 'redhat'
 supports 'oracle'
 supports 'amazon'
+supports 'opensuseleap'
```

Comment:
You might as well add `suse` here too.

Reply:
Done in e484c6d9dae478983ee498a5b4b78387f5e39caa